engine/src/main/java/org/terasology/rendering/dag/nodes/BloomBlurNode.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2016 MovingBlocks
```

Comment:
My apologies. There's something in IntelliJ about updating the copyright notice on commit. Should I have that enabled? Or should I just do it manually?